Max's Snake/Max's Snake/src/DynamicArray.c

```diff
@@ -0,0 +1,43 @@
+//
+//  DynamicArray.c
+//  Max's Snake
+//
+//  Created by Max Raulea on 06/04/2023.
+//
+
+
+#include "DynamicArray.h"
+#include <stdlib.h>
+
+const int MEM_BATCH_SIZE = 32;
+
+dArray* initArray(dArray *blocks){
```

Comment:
This confusing, same array as inout and, just in case of error, also returned.
Better to return a bool-like integer (0 or 1), as function result.